src/cli/src/consume/mod.rs

```diff
@@ -10,20 +10,19 @@ use std::convert::TryFrom;
 use tracing::{debug, trace, instrument};
 use structopt::StructOpt;
 use structopt::clap::arg_enum;
-use futures_lite::StreamExt;
+use fluvio_future::io::StreamExt;
 
 mod record_format;
 
 use fluvio::{Fluvio, PartitionConsumer, Offset, ConsumerConfig, FluvioError};
 use fluvio_sc_schema::ApiError;
 use fluvio::consumer::Record;
 
-use crate::ConsumerError;
+use crate::{Result, CliError};
```

Comment:
I think it is better to keep Error specific to Features instead of using CLI Error